sound/soc/sof/sof-pci-dev.c

```diff
@@ -283,9 +283,10 @@ static int sof_pci_probe(struct pci_dev *pci,
 
 	ret = snd_intel_dsp_driver_probe(pci);
 	if (ret != SND_INTEL_DSP_DRIVER_ANY &&
-	    ret != SND_INTEL_DSP_DRIVER_SOF)
+	    ret != SND_INTEL_DSP_DRIVER_SOF) {
```

Comment:
@plbossart while we're changing it can we move it to the previous line now that we have a bit more chars?

Reply:
done